cosmo_tester/test_suites/test_snapshots/snapshots_migration_from_321_and_331_to_34_test.py

```diff
@@ -0,0 +1,276 @@
+import json
```

Comment:
please add header file 
(e.g. https://github.com/cloudify-cosmo/cloudify-system-tests/blob/master/cosmo_tester/test_suites/test_blueprints/__init__.py#L1, should update the year)